app/electron/main.ts

```diff
@@ -313,14 +313,20 @@ function startElecron() {
         const storedAppVersion = store.get('app_version');
         if (!storedAppVersion) {
           store.set('app_version', appVersion);
-        } else if (storedAppVersion !== appVersion) {
+        } else if (parseInt(storedAppVersion as string) <= parseInt(appVersion)) {
```

Comment:
This will not do a correct comparison for the versions as `parseInt` returns `2` for `2.1.0`. There's a `semver` package that can be used for the comparison.